### DISCUSSION TITLE
how to use ng-matero SIDEBAR outside your project?

### DISCUSSION THREAD

**Discussion by AlonsoK28** [ng-matero/ng-matero#469] [Q&A] [answered]
I want to implement your **sidebar** into my project, how can I do it ☺

![image](https://user-images.githubusercontent.com/37458700/212815377-76cd1e65-222e-4a1a-9bba-ba448be4721c.png)

**Comment by nzbin** [comment, marked as answer]
Check the [sidebar](https://github.com/ng-matero/ng-matero/tree/master/src/app/theme/sidebar) component and copy it to your project.
https://github.com/ng-matero/ng-matero/blob/7891394134c1ef5727f6b973f628c11d91a571cf/src/app/theme/theme.module.ts#L7-L9
